src/libs/Navigation/types.ts

```diff
@@ -884,6 +884,7 @@ type AuthScreensParamList = SharedScreensParamList & {
     };
     [SCREENS.REPORT_AVATAR]: {
         reportID: string;
+        newGroupChat: string
```

Comment:
This should be boolean. Then in `src/libs/Navigation/linkingConfig/config.ts` add 

```js
parse: {
    newGroupChat: (newGroupChat: string) => newGroupChat === "true",
},
```